Makefile

```diff
@@ -73,7 +73,7 @@ test:
 # Run tests for examples
 
 test-examples:
-	python -m pytest -n auto --dist=loadfile -s -v ./examples/
+	python -m pytest -n auto --dist=loadfile -s -v ./examples/pytorch/
```

Comment:
We will just add `examples/tensorflow/` here. We can't run the tests from the root of the examples folder because it makes PyThon consider the `tensorflow` folder as the tensorflow library, which in turns makes everything fail.

Reply:
Does anybody ever use these targets? 

I never do because `-n auto` makes no sense. On my machine it starts 12 workers (12 cores) - and it will never complete the test suite with too many workers competing on gpus and consuming too much cpu RAM.